doc/Language/syntax.pod

```diff
@@ -395,6 +406,23 @@ You can also assign subroutines to variables.
 
     grammar Baz { }
 
+You can declare a unit of things without explicit curly brackets.
+
+    unit package Gar;
+    # ... stuff goes here instead of in {}'s
+
+=head3 Multi-dispatch declaration
+
+Subroutines can be declared with multiple signatures.
+
+    multi sub foo() { say "Hello!" }
+    multi sub foo($name) { say "Hello $name!" }
+
+Inside of a class, you can also declare multi-dispatch methods.
+
+    multi method greet { }
+    multi method greet(Str $name) { }
+
```

Comment:
somehow link to: http://docs.perl6.org/language/functions#Multi-dispatch "for more info" or you could just L<> the heading.

I plan to document `proto` there soon too.